//// [tests/cases/compiler/unreachableFlowAfterFinally.ts] ////

=== unreachableFlowAfterFinally.ts ===
function f() {
>f : () => number
>  : ^^^^^^^^^^^^

    let x = 100;
>x : number
>  : ^^^^^^
>100 : 100
>    : ^^^

    try {
        throw "WAT"
>"WAT" : "WAT"
>      : ^^^^^
    }
    catch (e) {
>e : any

    }
    finally {
        return x;
>x : number
>  : ^^^^^^
    }
}
